domain/domain.go

```diff
@@ -206,6 +215,32 @@ func (do *Domain) fetchSchemasWithTables(schemas []*model.DBInfo, m *meta.Meta,
 				continue
 			}
 			infoschema.ConvertCharsetCollateToLowerCaseIfNeed(tbl)
+			// check the repair mode.
+			if RepairMode {
+				inRepair := false
+				for _, tn := range RepairTableList {
+					// use dbName and tableName to specified a table.
+					if strings.ToLower(tn) == di.Name.L+"."+tbl.Name.L {
+						// this table is in repair table list.
+						inRepair = true
+						break
+					}
+				}
+				if inRepair {
+					// record the repaired table in Map.
+					if repairedDB, ok := RepairDBInfoMap[di.ID]; ok {
+						repairedDB.Tables = append(repairedDB.Tables, tbl)
+					} else {
+						// shallow copy the DB.
+						repairedDB := di.Copy()
+						// clean the tables and set repaired table.
+						repairedDB.Tables = []*model.TableInfo{}
+						repairedDB.Tables = append(repairedDB.Tables, tbl)
+						RepairDBInfoMap[di.ID] = repairedDB
```

Comment:
I think we need to check this map when executing `create table`. The scenario is as follows:  
There is a DDL operation that creates the table dbA_tblB before the operation of repairing dbA_tblB is started.

Reply:
Make sense, I will check this.